desktop.py

```diff
@@ -88,8 +88,7 @@ def setup_desktop_template(env: dict, opts: DesktopOpts, product: str, target_pl
         '--enable-maintainer-mode',
         '--with-tls=pthread',
         '--without-ikvm-native',
-        '--enable-btls',
-        '--enable-btls-lib'
```

Comment:
After reviewing all the changes, this seems to be the only part that has an effect on macOS builds. I think it's better to do this instead of reverting the entire thing.

So basically this line, then add this below:

```py
if target_platform == 'windows':
    CONFIGURE_FLAGS += [
        '--enable-btls-lib'
    ]
```

Or if we want to keep the change for Linux as well, then:

```py
if target_platform != 'osx':
    CONFIGURE_FLAGS += [
        '--enable-btls-lib'
    ]
```

Reply:
Well it turns out that this also breaks Windows builds, at least the MinGW ones: https://github.com/godotengine/godot/issues/61176

So for now I've simply reverted both #47 and #65 to go back to the tried and trusted configuration that we've been using for the past few months.